mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -35,13 +35,40 @@ interface TooltipProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should tooltip be shown on click
+   */
+  isShownOnClick?: boolean;
 }
 
 /**
  * Tooltip component
  */
 export const Tooltip = (props: PropsWithChildren<TooltipProps>) => {
   const [open, setOpen] = useState(false);
+  const isShownOnClick = props.isShownOnClick ?? true;
+
+  /**
+   * Toogle tooltip
+   */
+  const toggleOnClick = () => {
+    if (!isShownOnClick) {
+      return;
+    }
+
+    return setOpen((prevOpen) => !prevOpen);
+  };
+
+  /**
+   * Show tooltip
+   */
+  const showOnFocus = () => {
+    if (!isShownOnClick) {
+      return;
+    }
+    setTimeout(() => setOpen(true), 0);
```

Comment:
I suggest reverting my changes related to showing tooltips, and have someone create a separate task for me to handle tooltips on mobile, since the changes will affect the entire application